### PR TITLE
Fixing the linking issue and the subscription level view

### PR DESCRIPTION
**Pull request by StefArroyo**
## PR Checklist

* [ ] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

### If adding or updating templates:
* [ ] post a screenshot of templates and/or gallery changes
* [ ] ensure all steps have meaningful names
* [ ] ensure all parameters and grid columns have display names set so they can be localized
* [ ] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [ ] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [ ] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [ ] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

Fixed the linking in the workbook so that when a customer clicks on the Account link from the at scale view it should take them to the Insights (preview) link in their Azure Cosmos DB resource. 

EDIT:
Decided to keep the default subscriptions but just to mention that on the in the information bubble. 

Also changed the subscription ID option to include all subscriptions and not just the first few:
(Before)
image

(After)
image

### CODE REVIEW

File: Workbooks/CosmosDb/Overview/Overview.workbook [hunk 1]
```diff
@@ -1263,6 +1198,10 @@
       "name": "Azure Cosmos DB accounts"
     }
   ],
+  "fallbackResourceIds": [
+    "/subscriptions/d1eb41bc-1b7f-4404-bd2a-8568c222852d/resourceGroups/esarroyo-test/providers/Microsoft.DocumentDb/databaseAccounts/esarroyo-demo"
+  ],
   "workbookPin": "cosmosDB account metrics",
+  "fromTemplateId": "community-Workbooks/CosmosDb/Overview",
```
Comment (yves-chan) [inline Workbooks/CosmosDb/Overview/Overview.workbook:1205]: remove this

File: Workbooks/CosmosDb/Overview/Overview.workbook [hunk 2]
```diff
@@ -1263,6 +1198,10 @@
       "name": "Azure Cosmos DB accounts"
     }
   ],
+  "fallbackResourceIds": [
```
Comment (yves-chan) [inline Workbooks/CosmosDb/Overview/Overview.workbook:1201]: remove this

**Comment by StefArroyo** [comment]
@yves-chan Please take a look. I left the default subscriptions the same. Just added a note in the information bubble and fixed the linking issues!

File: Workbooks/CosmosDb/Overview/Overview.workbook [hunk 3]
```diff
@@ -220,7 +224,7 @@
           {
             "type": 1,
             "content": {
-              "json": "{Message}\r\n_Azure Cosmos DB accounts_\r\n<br />\r\n<br />"
+              "json": "{Message}\r\n_Azure Cosmos DB Accounts_\r\n<br />\r\n<br />"
```
Comment (yves-chan) [inline Workbooks/CosmosDb/Overview/Overview.workbook:227]: just curious, does 'Accounts' have to be capitalized? Might be marketing thing that we left accounts lowercase on purpose

**Review by yves-chan — APPROVED** [review]

**Pull request merged**